README.md

```diff
@@ -25,6 +25,7 @@ This client library supports all of Stytch's live products:
   - [x] [Email passcodes](https://stytch.com/docs/api/send-otp-by-email)
   - [x] [Session Management (Beta)](https://stytch.com/docs/api/sessions-overview)
   - [x] [WebAuthn (Beta)](https://stytch.com/docs/api/webauthn-overview)
+  - [x] [User Management (Beta)](https://stytch.com/docs/api/users)
```

Comment:
I think I'm capitulating here, people seem to prefer sentence case for product names as well, e.g. User management.

```suggestion
  - [x] [User management (Beta)](https://stytch.com/docs/api/users)
```